packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1533,6 +1539,155 @@ The following response is only a subset of properties for brevity.
 }
 ```
 
+## Log Stream Schema operations
+
+<ApiLifecycle access="ea" />
+
+> **Note:** The **Log Streaming** Early Access feature must be enabled. See [Manage Early Access and Beta features](https://help.okta.com/okta_help.htm?id=ext_Manage_Early_Access_features) for more information on Okta's Self-Service Feature Manager.
+
+### Get Log Stream Schema
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="get" url="/api/v1/meta/schemas/logStream/${typeId}" />
+
+Fetches the schema for a Log Stream type
+
+See [Log Streaming API](/docs/reference/api/log-streaming) for examples of Log Stream objects.
+
+##### Request parameters
+
+N/A
```

Comment:
Should we have a description for `${typeId}`, similar to [Get App User Schema]? 
Or at least tell the user that `typeId` == `/logStream/aws_eventbridge`?

Reply:
added a sentence above